managed/ui/src/components/common/associatedUniverse/AssociatedUniverse.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { YBModal } from '../forms/fields';
+import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
+import { Link } from 'react-router';
+import './AssociatedUniverse.scss';
+
+function AssociatedUniverse(props) {
+  const { onHide, associatedUniverses, visible, title } = props;
+  /**
+   * Returns the decorated status.
+   * @param universeStatus status of universe.
+   */
+  const modifyStatus = (item,row) => {
+    const universeStatus = row.universePaused
+      ? 'Paused'
+      : row.updateSucceeded && !row.updateInProgress
+      ? 'Ready'
+      : 'Error';
+    // TODO: Add case for pause status.
+    switch (universeStatus) {
+      case 'Ready':
+        return (
+          <div className="universe-status associated-universe-status good">
+            <i className="fa fa-check-circle associated-universe-icon" />
+            {universeStatus && <span>{universeStatus}</span>}
+          </div>
+        );
+      case 'Error':
+        return (
+          <div className="universe-status associated-universe-status bad">
+            <i className="fa fa-warning associated-universe-icon" />
+            {universeStatus && <span>{universeStatus}</span>}
+          </div>
+        );
+      case 'Paused':
+        return (
+          <div className="universe-status associated-universe-status paused">
+            <i className="fa fa-pause-circle-o associated-universe-icon" />
+            {universeStatus && <span>{universeStatus}</span>}
+          </div>
+        );
+    }
+  };
+
+  /**
+   * Returns the universe name with router link.
+   * @param universeName - Name of the universe.
+   */
+  const getUniverseLink = (universeName, row) => {
+    return (
+      <div>
+        <Link to={`/universes/${row.uuid}`}>{universeName}</Link>
+      </div>
+    );
+  };
+
+  return (
+    <YBModal
+      visible={visible}
+      onHide={onHide}
+      submitLabel={'Close'}
+      onFormSubmit={onHide}
+      title={`Universes using this ${title}`}
+    >
+      {associatedUniverses?.length ? (
+        <div>
+          <BootstrapTable
+            data={associatedUniverses}
+            className="backup-list-table middle-aligned-table"
+          >
+            <TableHeaderColumn dataField="uuid" hidden={true} isKey={true}>
+              UUID
+            </TableHeaderColumn>
+            <TableHeaderColumn
+              dataField="name"
+              columnClassName="no-border name-column"
+              className="no-border"
+              width="80%"
+              dataFormat={getUniverseLink}
```

Comment:
Can we make the entire row clickable instead of just 1 column?

Reply:
This is consistence with the design in this application for e.g. Check the nodes page of any created universe there we have the universe name is decorated as a link.